### DISCUSSION TITLE
New pub dev version please :)

### DISCUSSION THREAD

**Discussion by EArminjon** [cmdrootaccess/another-flushbar#113] [General]
Somes PR got merged recently, do you plan to release soon a new pub dev version ?

PR like : 
https://github.com/cmdrootaccess/another-flushbar/pull/109
https://github.com/cmdrootaccess/another-flushbar/pull/86

@cmdrootaccess

**Comment by cmdrootaccess** [comment]
> Somes PR got merged recently, do you plan to release soon a new pub dev version ?
> 
> 
> 
> PR like : 
> 
> https://github.com/cmdrootaccess/another-flushbar/pull/109
> 
> https://github.com/cmdrootaccess/another-flushbar/pull/86
> 
> 
> 
> @cmdrootaccess 

yes i will publish them to a dev version later on today

**Comment by cmdrootaccess** [comment]
published https://pub.dev/packages/another_flushbar/versions/1.12.31-dev.1. @EArminjon
